internal/provider/resource_user.go

```diff
@@ -43,6 +43,12 @@ func resourceUser() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"active": {
+				Description: "Active",
+				Type:        schema.TypeBool,
+				Required:    false,
```

Comment:
Seems like `Optional: true` instead of `Required: false` is needed here. Then the tests are running fine.

Reply:
Thanks, I will update the code. Any idea what's the difference between the 2?

Reply:
Seems like they are semantically different: `Required: true` will express that a value *needs* to be provided, `Optional: true` says that it *can* be provided. The logical negation of each attribute however does not result in the same result: `Required: false` and `Optional: false` seem to be the "default" and have no effect. Only an explicit value of `true` for each attribute will trigger the respective behaviour.

Take this with a pinch of salt: I'm no expert in writing Terraform providers either, but that seems to be what I'm reading from the Schema documentation.